Automattic/jetpack
# Jetpack: Show the IDC screen immediately after leaving Safe Mode

When you leave the "Stay in Safe Mode" state, Jetpack Dashboard will show up but the IDC screen will not.
Another page reload makes things right, but it should happen immediately.

## Proposed changes:
* Fix the bug where IDC screen doesn't show up immediately after you leave the "Confirmed Safe Mode".

### Other information:


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Connect a Jetpack site, use debugging tools to trigger an IDC. Deactivate the "WPCOM toolbar" if active.
2. Go to Jetpack Dashboard, confirm you see the IDC screen and don't see the dashboard.
3. Click "Stay in Safe Mode", confirm that Jetpack Dashboard showed up immediately after page reload, and the orange "Jetpack Safe Mode" button appeared in the top toolbar.
4. Click that orange button. Confirm that page reloaded and now you see the IDC screen again without having to reload page again manually.
